<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Research grants | MI²</title>
  <meta name="description" content="MI2" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Research grants | MI²" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="/images/cover.png" />
  <meta property="og:description" content="MI2" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Research grants | MI²" />
  
  <meta name="twitter:description" content="MI2" />
  <meta name="twitter:image" content="/images/cover.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="papers.html"/>
<link rel="next" href="thesis-proposals.html"/>
<script src="libs/jquery/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections/anchor-sections.js"></script>
<style type="text/css">
    .book .book-body .body-inner {
        position: absolute;
        top: -50px;
    }
</style>

<link rel='shortcut icon' type='image/x-icon' href='/favicon.ico' />



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="css/styles.css" type="text/css" />
<link rel="stylesheet" href="css/typography.css" type="text/css" />
<link rel="stylesheet" href="css/toc.css" type="text/css" />
<link rel="stylesheet" href="css/variables.css" type="text/css" />
<link rel="stylesheet" href="css/two-columns-layout.css" type="text/css" />
<link rel="stylesheet" href="css/icons.css" type="text/css" />
<link rel="stylesheet" href="css/team-layout.css" type="text/css" />
<link rel="stylesheet" href="css/research-grants-layout.css" type="text/css" />
<link rel="stylesheet" href="css/images.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>MI².AI</a>
<ul>
<li class="chapter" data-level="" data-path="the-team.html"><a href="the-team.html"><i class="fa fa-check"></i>The Team</a></li>
<li class="chapter" data-level="" data-path="open-positions.html"><a href="open-positions.html"><i class="fa fa-check"></i>Open Positions</a>
<ul>
<li class="chapter" data-level="" data-path="open-positions.html"><a href="open-positions.html#mi2-policy-on-benefits-and-obligations"><i class="fa fa-check"></i>MI2 policy on benefits and obligations</a></li>
<li class="chapter" data-level="" data-path="open-positions.html"><a href="open-positions.html#research-software-engineer"><i class="fa fa-check"></i>Research Software Engineer</a></li>
<li class="chapter" data-level="" data-path="open-positions.html"><a href="open-positions.html#how-to-write-a-good-motivation-letter"><i class="fa fa-check"></i>How to write a good motivation letter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="seminars.html"><a href="seminars.html"><i class="fa fa-check"></i>Seminars</a></li>
<li class="chapter" data-level="" data-path="papers.html"><a href="papers.html"><i class="fa fa-check"></i>Papers</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html"><i class="fa fa-check"></i>Research grants</a>
<ul>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#demeter-2024-2028"><i class="fa fa-check"></i>DeMeTeR: 2024-2028</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#pineberry-2024-2025"><i class="fa fa-check"></i>PINEBERRY: 2024-2025</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#pvstatem-2023-2027"><i class="fa fa-check"></i>PvSTATEM 2023-2027</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#gliomai-2024"><i class="fa fa-check"></i>GliomAI 2024</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#pineapple-2023-2025"><i class="fa fa-check"></i>PINEAPPLE 2023-2025</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#ares-2022-2026"><i class="fa fa-check"></i>ARES 2022-2026</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#darling-2022-2024"><i class="fa fa-check"></i>DARLING 2022-2024</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#x-lungs-2021-2024"><i class="fa fa-check"></i>X-LUNGS 2021-2024</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#homer-2020-2025"><i class="fa fa-check"></i>HOMER 2020-2025</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#decovid-2020-2022"><i class="fa fa-check"></i>DeCoviD 2020-2022</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#dalex-2018-2022"><i class="fa fa-check"></i>DALEX 2018-2022</a></li>
<li class="chapter" data-level="" data-path="research-grants.html"><a href="research-grants.html#mlgensig-2017-2021"><i class="fa fa-check"></i>MLGenSig 2017-2021</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="thesis-proposals.html"><a href="thesis-proposals.html"><i class="fa fa-check"></i>Thesis proposals</a>
<ul>
<li class="chapter" data-level="" data-path="thesis-proposals.html"><a href="thesis-proposals.html#red-teaming-ai-models"><i class="fa fa-check"></i>Red Teaming AI models</a></li>
<li class="chapter" data-level="" data-path="thesis-proposals.html"><a href="thesis-proposals.html#xai-against-cancer"><i class="fa fa-check"></i>XAI against Cancer</a></li>
<li class="chapter" data-level="" data-path="thesis-proposals.html"><a href="thesis-proposals.html#xai-for-space"><i class="fa fa-check"></i>XAI for Space</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="contact.html"><a href="contact.html"><i class="fa fa-check"></i>Contact</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="mi2redteam.html"><a href="mi2redteam.html"><i class="fa fa-check"></i>MI²RedTeam</a></li>
<li class="chapter" data-level="" data-path="mi2cancer.html"><a href="mi2cancer.html"><i class="fa fa-check"></i>MI²<del>Cancer</del></a></li>
<li class="chapter" data-level="" data-path="mi2space.html"><a href="mi2space.html"><i class="fa fa-check"></i>MI²Space</a></li>
<li class="chapter" data-level="" data-path="mi2betabit.html"><a href="mi2betabit.html"><i class="fa fa-check"></i>MI²BetaBit</a></li>
<li class="chapter" data-level="" data-path="mi2solutions.html"><a href="mi2solutions.html"><i class="fa fa-check"></i>MI²Solutions</a></li>
<li class="divider"></li>
<li class="social-icons"> 
  <a href="https://github.com/MI2DataLab"> <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg"> <path fill-rule="evenodd" clip-rule="evenodd" d="M8.1951 0.0301564C6.24962 0.0281791 4.36703 0.719084 2.88468 1.97906C1.40234 3.23904 0.417131 4.98573 0.105636 6.90611C-0.205859 8.82649 0.176714 10.795 1.18479 12.459C2.19286 14.1229 3.76055 15.3735 5.6069 15.9865C6.01398 16.0619 6.16475 15.8056 6.16475 15.5895C6.16475 15.3734 6.16475 14.8809 6.16475 14.1974C3.88814 14.7 3.40567 13.1018 3.40567 13.1018C3.25397 12.6065 2.93169 12.181 2.49603 11.9007C1.75726 11.3981 2.55634 11.3981 2.55634 11.3981C2.81531 11.4347 3.06256 11.5297 3.27936 11.676C3.49617 11.8223 3.67686 12.016 3.80772 12.2424C3.91892 12.4441 4.06883 12.6218 4.24887 12.7653C4.42891 12.9088 4.63552 13.0154 4.85684 13.0789C5.07817 13.1424 5.30986 13.1616 5.53861 13.1354C5.76737 13.1091 5.98869 13.0379 6.18987 12.9259C6.22141 12.5125 6.39965 12.1239 6.69243 11.8303C4.87315 11.6243 2.96341 10.9207 2.96341 7.78471C2.95092 6.97206 3.25142 6.18572 3.80269 5.58851C3.55667 4.88199 3.58538 4.10879 3.88311 3.42246C3.88311 3.42246 4.57162 3.20133 6.13459 4.26174C7.47673 3.89324 8.89337 3.89324 10.2355 4.26174C11.7985 3.20133 12.482 3.42246 12.482 3.42246C12.7828 4.10781 12.8133 4.88159 12.5674 5.58851C13.1187 6.18572 13.4192 6.97206 13.4067 7.78471C13.4067 10.9308 11.4919 11.6193 9.66761 11.8052C9.86322 12.0019 10.0143 12.2383 10.1105 12.4984C10.2068 12.7585 10.246 13.0363 10.2255 13.3129C10.2255 14.4085 10.2255 15.293 10.2255 15.5594C10.2255 15.8257 10.3712 16.0318 10.7883 15.9564C12.6289 15.3375 14.1897 14.085 15.1925 12.4222C16.1953 10.7593 16.5748 8.79434 16.2633 6.87768C15.9518 4.96103 14.9696 3.21735 13.4918 1.95765C12.0141 0.697949 10.1369 0.0041503 8.1951 0V0.0301564Z" fill="black"/> </svg> </a> 
  <a href="https://www.linkedin.com/company/mi2datalab"> <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M16.2913 15.2715V9.73481C16.2913 7.02356 15.7278 4.94019 12.6007 4.94019C11.1075 4.94019 10.0933 5.76783 9.67071 6.56693H9.61437V5.19704H6.65625V15.2715H9.75524V10.2771C9.75524 8.96424 10.0088 7.67997 11.6146 7.67997C13.1923 7.67997 13.2205 9.19256 13.2205 10.3627V15.2715H16.2913Z" fill="black"/> <path d="M0.962891 4.94019H4.46654V15.2715H0.962891V4.94019Z" fill="black"/> <path d="M2.71471 0C1.75668 0 0.962891 0.786075 0.962891 1.79674C0.962891 2.80741 1.72931 3.59348 2.71471 3.59348C3.67274 3.59348 4.46654 2.77934 4.46654 1.79674C4.43917 0.786075 3.67274 0 2.71471 0Z" fill="black"/> </svg> </a>
</li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">MI²</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="research-grants" class="section level2 unnumbered hasAnchor">
<h2>Research grants<a href="research-grants.html#research-grants" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<script>
    document.body.classList.add("research-grants-page")
    document.querySelector(".page-inner section > *:first-child").classList.add("research-grants-layout")
</script>
<div id="demeter-2024-2028" class="section level3 unnumbered hasAnchor">
<h3>DeMeTeR: 2024-2028<a href="research-grants.html#demeter-2024-2028" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="demeter-interpreting-diffusion-models-through-representations" class="section level4 unnumbered hasAnchor">
<h4>DeMeTeR: Interpreting Diffusion Models Through Representations<a href="research-grants.html#demeter-interpreting-diffusion-models-through-representations" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-demeter.png" /></p>
<p>Diffusion models have been the latest revolution in the domain of generative modelling in computer vision, surpassing the capabilities of long-reigning generative adversarial networks , and are currently being adapted to multiple other domains and modalities. However, we still lack an in-depth understanding of their inner workings from both an empirical and theoretical standpoint.</p>
<p>Considering that, the main goals of the DeMeTeR project are:</p>
<ol style="list-style-type: decimal">
<li>to broaden the practical and theoretical understanding of diffusion-specific latent representations and architecture-specific internal representations of diffusion models,</li>
<li>to develop novel methods of manipulating these representations that allow for enhancing safety and explainability of deep learning models</li>
</ol>
<p><strong>Work on this project is financially supported by the Polish National Science Centre PRELUDIUM BIS grant 2023/50/O/ST6/00301.</strong></p>
</div>
</div>
<div id="pineberry-2024-2025" class="section level3 unnumbered hasAnchor">
<h3>PINEBERRY: 2024-2025<a href="research-grants.html#pineberry-2024-2025" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="pineberry-explainable-robust-and-secure-ai-for-demistyfying-space-operations" class="section level4 unnumbered hasAnchor">
<h4>PINEBERRY: Explainable, Robust and Secure AI for demistyfying Space Operations<a href="research-grants.html#pineberry-explainable-robust-and-secure-ai-for-demistyfying-space-operations" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-pineberry.png" /></p>
<p>In the PINEBERRY project (ExPlaINablE, RoBust And SEcuRe AI FoR DemYstifying Space Mission Operations), we will address the important research gap of lack of “trust” into (deep) machine learning algorithms for space mission operations, through tackling real-life downstream for the A2I roadmap using deep and classic machine learning algorithms empowered by new security and explainable AI (XAI) techniques. We believe that PINEBERRY will be an important step toward not only “uncovering the magic” behind deep learning algorithms (hence building trust in them in downstream tasks), but also in showing that XAI techniques can be effectively utilized to improve such data-driven algorithms (both classic and deep machine learning-powered), ultimately leading to better algorithms. Finally, we will put special effort into (i) unbiasing the validation of existing and emerging algorithms through ensuring their full reproducibility (both at the algorithm and at the data level), and (ii) ensuring security of algorithms on different levels of data, models, and system.</p>
<p>To this end we develop a catalogue for issues relevant to Security and Explainability that will guide validation of ML/AI models used in SO. This project is conducted in collaboration with KP Labs company.</p>
<p><strong>Work on this project is financially supported by European Space Agency grant ESA 4000144194/23/D/BL.</strong></p>
</div>
</div>
<div id="pvstatem-2023-2027" class="section level3 unnumbered hasAnchor">
<h3>PvSTATEM 2023-2027<a href="research-grants.html#pvstatem-2023-2027" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="pvstatem-serological-testing-and-treatment-for-p.-vivax-from-a-cluster-randomised-trial-in-ethiopia-and-madagascar-to-a-mobile-technology-supported-intervention" class="section level4 unnumbered hasAnchor">
<h4>PvSTATEM: Serological testing and treatment for P. Vivax: from a cluster-randomised trial in Ethiopia and Madagascar to a mobile-technology supported intervention<a href="research-grants.html#pvstatem-serological-testing-and-treatment-for-p.-vivax-from-a-cluster-randomised-trial-in-ethiopia-and-madagascar-to-a-mobile-technology-supported-intervention" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/mi2-pvstatem.png" /></p>
<p>The PvSTATEM project aims to demonstrate the efficacy and the community acceptability of P. vivax Serological Testing and Treatment (PvSeroTAT), a new
intervention for the control and elimination of malaria, in cluster-randomised trials in Ethiopia and Madagascar. The project will also innovate new mobile
technologies for the efficient implementation of PvSeroTAT in settings beyond clinical trials. The PvSeroTAT intervention includes a serological diagnostic
test that measures antibodies to multiple P. vivax antigens and informs an individual-level treatment decision. However, the results from serological tests can also inform population-level surveillance of malaria. In this Hop-on project, mathematical models, machine learning tools, and digital technologies will be developed so that data generated by the clinical trials in Ethiopia and Madagascar can inform national malaria surveillance programs.</p>
<p><strong>Work on this project is financially supported by the HORIZON grant HORIZON-WIDERA-2022-ACCESS-07-01.</strong></p>
</div>
</div>
<div id="gliomai-2024" class="section level3 unnumbered hasAnchor">
<h3>GliomAI 2024<a href="research-grants.html#gliomai-2024" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="gliomai-artificial-intelligence-for-radiogenomic-atlas-of-gliomas" class="section level4 unnumbered hasAnchor">
<h4>GliomAI: Artificial Intelligence for Radiogenomic Atlas of Gliomas<a href="research-grants.html#gliomai-artificial-intelligence-for-radiogenomic-atlas-of-gliomas" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/mi2-gliomai.png" /></p>
<p>The new 2021 WHO classification of brain tumours places more emphasis than before on genetic variation in the classification of tumour lesions. However, invasive procedures are required for genetic diagnosis, which pose risks to patients and limit access to molecular profiling. Radiomics, a non-invasive approach, allows the analysis of tumour features using imaging data such as magnetic resonance imaging (MRI), which is used to extract computational independent variables. This approach allows the analysis of heterogeneity, spatial relationships and textural patterns that characterise different tumour phenotypes, however, may not be graspable by human perception. The correlation of such computational variables obtained with genetic findings is called radiogenomics.</p>
<p>Multidimensional datasets play a key role in the development of the field of radiogenomics. However, in order to do so, it is necessary to delineate regions of interest within imaging studies - so-called masks - which are ultimately used to extract computational variables. In this project, we plan to develop a novel radiomic database containing not only clinical, genetic and imaging data, but also the previously mentioned segmentation masks of gliomas and their immediate surroundings. To this end, an interdisciplinary research team will be formed, benefiting from the synergistic impact of the two units involved in the project at our Universities.</p>
<p><strong>Work on this project is financially supported by Warsaw Medical University and Warsaw University of Technology within the Collaboration Initiative Programme WUM_PW INTEGRA 1.</strong></p>
</div>
</div>
<div id="pineapple-2023-2025" class="section level3 unnumbered hasAnchor">
<h3>PINEAPPLE 2023-2025<a href="research-grants.html#pineapple-2023-2025" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="pineapple-explainable-ai-for-hyperspectral-image-analysis" class="section level4 unnumbered hasAnchor">
<h4>PINEAPPLE: Explainable AI for hyperspectral image analysis<a href="research-grants.html#pineapple-explainable-ai-for-hyperspectral-image-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-pineapple.png" /></p>
<p>In the PINEAPPLE project (exPlaINablE Ai for hyPersPectraL imagE analysis), we will address the important research gap of lack of “trust” into (deep) machine learning algorithms for EO, through tackling two real-life EO downstream tasks (estimating soil parameters from HSI and detecting methane in such imagery) using new deep and classic machine learning algorithms empowered by new explainable AI (XAI) techniques. We believe that PINEAPPLE will be an important step toward not only “uncovering the magic” behind deep learning algorithms (hence building trust in them in EO downstream tasks), but also in showing that XAI techniques can be effectively utilized to improve such data-driven algorithms (both classic and deep machine learning-powered), ultimately leading to better algorithms.</p>
<p>Finally, we will put special effort into:</p>
<ol style="list-style-type: decimal">
<li>unbiasing the validation of existing and emerging algorithms through ensuring their full reproducibility (both at the algorithm and at the data level), and</li>
<li>understanding &amp; improving the generalization of such algorithms when fundamentally different data is used for testing (e.g., noisy, with simulated other atmospheric conditions, captured in different area/time, and so forth)</li>
</ol>
<p><strong>Work on this project is financially supported by European Space Agency grant ESA AO/1-11524/22/I-DT.</strong></p>
</div>
</div>
<div id="ares-2022-2026" class="section level3 unnumbered hasAnchor">
<h3>ARES 2022-2026<a href="research-grants.html#ares-2022-2026" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="ares-attack-resistant-explanations-toward-secure-and-trustworthy-ai" class="section level4 unnumbered hasAnchor">
<h4>ARES: Attack-resistant Explanations toward Secure and trustworthy AI<a href="research-grants.html#ares-attack-resistant-explanations-toward-secure-and-trustworthy-ai" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/mi2-ares.png" /></p>
<p>Machine learning explainability, fairness, robustness, and security are key elements of trustworthy Artificial
Intelligence, an area of strategic importance. In this context, the main goals of the ARES project are:</p>
<ol style="list-style-type: decimal">
<li>Develop adversarial attacks on state-of-the-art explanations to investigate vulnerabilities and
limitations of the existing explainability and fairness approaches in machine learning.</li>
<li>Introduce novel robust explanations that are stable against manipulation and intuitive to evaluate.</li>
</ol>
<p>Achieving the first goal primarily impacts various domains of research, which currently use (and explain) black-box models for knowledge discovery and decision-making, by highlighting vulnerabilities and limitations of their explanations. Achieving the second goal impacts more the broad machine learning domain as it aims at improving state-of-the-art by introducing robust explanations toward secure and trustworthy AI.</p>
<p><strong>Work on this project is financially supported by the Polish National Science Centre PRELUDIUM BIS grant 2021/43/O/ST6/00347.</strong></p>
</div>
</div>
<div id="darling-2022-2024" class="section level3 unnumbered hasAnchor">
<h3>DARLING 2022-2024<a href="research-grants.html#darling-2022-2024" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="darling-deep-analysis-of-regulations-with-language-inference-network-analysis-and-institutional-grammar" class="section level4 unnumbered hasAnchor">
<h4>DARLING: Deep Analysis of Regulations with Language Inference, Network analysis and institutional Grammar<a href="research-grants.html#darling-deep-analysis-of-regulations-with-language-inference-network-analysis-and-institutional-grammar" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-darling.png" /></p>
<p><strong>Aim of the project</strong></p>
<p>Developing the tools for automated analysis of content of legal documents leveraging Natural Language Processing, that will help understand the dynamic of change in public policies and variables influencing those changes.</p>
<p>Those tools will be firstly used to analyse the case of development of policy subsystem regulating usage of AI in the European Union.</p>
<p><strong>Specific goals of the project</strong></p>
<ol style="list-style-type: decimal">
<li>Developing and evaluating multilingual models for issue classification for legal and public policy documents.</li>
<li>Developing embedding-based topic modeling methods for legal and public policy documents suited for analysis of change of the topics between documents.</li>
<li>Institutional grammar based analysis of changes in topics between different public policy documents, regulations and public consultation documents.</li>
<li>Agent-based models predicting diffusion of issues in public policy documents.</li>
</ol>
<p><strong>Methodology</strong></p>
<p>The core of the DARLING project is the issues and topic analysis in documents connected with regulations development using NLP tools. Issues analysis shall allow tracking how different options of AI operationalisation, ways the AI-connected threats are perceived as well as ideas regarding AI regulations are shared among three different types of texts: scientific, expert and legal ones. The extracted issues will then be subject to complex networks analysis and institutional grammar approach. The network analysis, backed by agent-based modeling, will be used to examine the flow of issues among the documents based on their vector-formed characteristics. On the other hand, the Institutional Grammar (IG) will be used to analyze the modality of issues, e.g., the tendency to regulate a specific aspect of AI in a given issue, its deontic character or its conditionality.</p>
<p>In result the DARLING project will effect in the development of new methods to analyze legal documents connected to regulation based on deep text processing and links among the documents. An inter-institutional and interdisciplinary team of computer, political sciences and physics of complex systems scientists will elaborate new machine learning approaches to examine the regulation corpora, issues recognition, issues analysis by the means of IG as well as propose new methods of modeling the flow/changes of regulations based on complex networks tools.</p>
</div>
</div>
<div id="x-lungs-2021-2024" class="section level3 unnumbered hasAnchor">
<h3>X-LUNGS 2021-2024<a href="research-grants.html#x-lungs-2021-2024" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="x-lungs-responsible-artificial-intelligence-for-lung-diseases" class="section level4 unnumbered hasAnchor">
<h4>X-LUNGS: Responsible Artificial Intelligence for Lung Diseases<a href="research-grants.html#x-lungs-responsible-artificial-intelligence-for-lung-diseases" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/projects_xlungs.jpg" /></p>
<p>The aim of the project is to support the process of identification of lesions visible on CT and lung x-rays. We intend to achieve this goal by building an information system based on artificial intelligence (AI) that will support the radiologist’s work by enriching the images with additional information.</p>
<p>The unique feature of the proposed system is a trustworthy artificial intelligence module that:</p>
<ul>
<li>will reduce the image analysis time needed to detect lesions,</li>
<li>will make the image evaluation process more transparent,</li>
<li>will provide image and textual explanations indicating the rationale behind the proposed recommendation,</li>
<li>will be verified for effective collaboration with the radiologist.</li>
</ul>
<p><strong>Work on this project is financially supported from the INFOSTRATEG-I/0022/2021-00 grant funded by Polish National Centre for Research and Development (NCBiR).</strong></p>
</div>
</div>
<div id="homer-2020-2025" class="section level3 unnumbered hasAnchor">
<h3>HOMER 2020-2025<a href="research-grants.html#homer-2020-2025" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="homer-human-oriented-automated-machine-learning" class="section level4 unnumbered hasAnchor">
<h4>HOMER: Human Oriented autoMated machinE leaRning<a href="research-grants.html#homer-human-oriented-automated-machine-learning" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-homer.png" /></p>
<p>One of the biggest challenges in the state-of-the-art machine learning is dealing with the complexity of predictive models. Recent techniques like deep neural networks, gradient boosting or random forests create models with thousands or even millions of parameters. This makes decisions generated by these black-box models completely opaque. Model obscurity undermines trust in model decisions, hampers model debugging, blocks model auditability, exposes models to problems with concept drift or data drift.</p>
<p>Recently, there has been a huge progress in the area of model interpretability, which results in the first generation of model explainers, methods for better understanding of factors that drive model decisions. Despite this progress, we are still far from methods that provide deep explanations, confronted with domain knowledge that satisfies our ,,Right to explanation’’ as listed in the General Data Protection Regulation (GDPR).</p>
<p>In this project I am going to significantly advance next generation of explainers for predictive models. This will be a disruptive change in the way how machine learning models are created, deployed, and maintained. Currently to much time is spend on handcrafted models produced in a tedious and laborious try-and-error process. The proposed Human-Oriented Machine Learning will focus on the true bottleneck in development of new algorithms, i.e. on model-human interfaces.</p>
<p>The particular directions I consider are (1) developing an uniform grammar for visual model exploration, (2) establishing a methodology for contrastive explanations that describe similarities and differences among different models, (3) advancing a methodology for non-additive model explanations, (4) creating new human-model interfaces for effective communication between models and humans, (5) introducing new techniques for training of interpretable models based on elastic surrogate black-box models, (6) rising new methods for automated auditing of fairness, biases and performance of predictive models.</p>
<p><strong>Work on this project is financially supported from the SONATA BIS grant 2019/34/E/ST6/00052 funded by Polish National Science Centre (NCN).</strong></p>
</div>
</div>
<div id="decovid-2020-2022" class="section level3 unnumbered hasAnchor">
<h3>DeCoviD 2020-2022<a href="research-grants.html#decovid-2020-2022" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="decovid-detection-of-covid-19-related-markers-of-pulmonary-changes-using-deep-neural-networks-models-supported-by-explainable-artificial-intelligence-and-cognitive-compressed-sensing" class="section level4 unnumbered hasAnchor">
<h4>DeCoviD: Detection of Covid-19 related markers of pulmonary changes using Deep Neural Networks models supported by eXplainable Artificial Intelligence and Cognitive Compressed Sensing<a href="research-grants.html#decovid-detection-of-covid-19-related-markers-of-pulmonary-changes-using-deep-neural-networks-models-supported-by-explainable-artificial-intelligence-and-cognitive-compressed-sensing" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-decovid.png" /></p>
<p><br />
<br /></p>
<p>Covid-19 is an infectious respiratory disease. A coronavirus infection leaves permanent ramifications in the respiratory system and beyond. In this situation, tools supporting diagnosis and assessment of lung damage after infection and during Covid-19 treatment are crucial. Preliminary results of analysis of CT images and lung xrays suggest that they can help to quickly assess even asymptomatic cases and facilitate prognosis of response to treatment. There are also reports of usefulness of ultrasound images.</p>
<p>The aim of the DeCoviD project is to develop methods and tools to support radiologists in the assessment of lung imaging data for the occurrence of changes caused by Covid-19 disease. The developed solution will allow to automate the identification of pathological changes and will support the diagnosis of coexisting lung diseases as well as diseases of other organs visible on chest images. It will also allow to quantify the severity of lung damage caused by the disease</p>
<p>Responsible decision support for radiologists requires models based on interpretable features. Such features will be stored in a hybrid knowledge base powered by two research teams from WUT, working on the basis of two, seemingly opposite, paradigms of image data analysis. The eXplainable Artificial Intelligence (XAI) team will use trained deep networks to automatically extract features that are essential for effective disease detection. Cognitive Compressed Sensing (CCS) will build a set of interpretable semantic features using sparse cognitive representations agreed with a group of cooperating radiologists. Combining these two approaches will achieve high effectiveness of the constructed models, combined with high transparency, clarity and stability of the solution.</p>
<p>The DeCoviD project is a part of a broader strategy of competence development in the area of deep learning + XAI + medical applications at the Warsaw University of Technology.</p>
<p>More information: <a href="https://github.com/MI2DataLab/DeCoviD" class="uri">https://github.com/MI2DataLab/DeCoviD</a>.</p>
<p><strong>Work on this project is financially supported by the IDUB against COVID PW.</strong></p>
</div>
</div>
<div id="dalex-2018-2022" class="section level3 unnumbered hasAnchor">
<h3>DALEX 2018-2022<a href="research-grants.html#dalex-2018-2022" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="dalex-descriptive-and-model-agnostic-local-explanations" class="section level4 unnumbered hasAnchor">
<h4>DALEX: Descriptive and model Agnostic Local EXplanations<a href="research-grants.html#dalex-descriptive-and-model-agnostic-local-explanations" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-dalex.png" /></p>
<p><strong>Research project objectives.</strong> Black boxes are complex machine learning models, for example deep neural network, an ensemble of trees of high-dimensional regression model. They are commonly used due to they high performance. But how to understand the structure of a black-box, a model in which decision rules are too cryptic for humans? The aim of the project is to create a methodology for such exploration. To address this issue we will develop methods, that: (1) identify key variables that mostly determine a model response, (2) explain a single model response in a compact visual way through local approximations, (3) enrich model diagnostic plots.</p>
<p><strong>Research project methodology.</strong> This project is divided into three subprojects - local approximations od complex models (called LIVE), explanations of particular model predictions (called EXPLAIN) and conditional explanations (called CONDA).</p>
<p><strong>Expected impact on the development of science.</strong> Explanations of black boxes have fundamental implications for the field of predictive and statistical modelling. The advent of big data forces imposes usage of black boxes that are easily able to overperform classical methods. But the high performance itself does not imply that the model is appropriate. Thus, especially in applications to personalized medicine or some regulated fields, one should scrutinize decision rules incorporated in the model. New methods and tools for exploration of black-box models are useful for quick identification of problems with the model structure and increase the interpretability of a black-box</p>
<p><strong>Work on this project is financially supported from the OPUS grant 2017/27/B/ST6/01307 funded by Polish National Science Centre (NCN).</strong></p>
</div>
</div>
<div id="mlgensig-2017-2021" class="section level3 unnumbered hasAnchor">
<h3>MLGenSig 2017-2021<a href="research-grants.html#mlgensig-2017-2021" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="mlgensig-machine-learning-methods-for-building-of-integrated-genetic-signatures" class="section level4 unnumbered hasAnchor">
<h4>MLGenSig: Machine Learning Methods for building of Integrated Genetic Signatures<a href="research-grants.html#mlgensig-machine-learning-methods-for-building-of-integrated-genetic-signatures" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="images/project-mlgensig.png" /></p>
<p><strong>Research project objectives.</strong> The main scientific goal of this project is to develop a methodology for integrated genetic signatures based on data from divergent high-throughput techniques used in molecular biology. Integrated signatures base on ensembles of signatures for RNA-seq, DNA-seq, data as well for methylation profiles and protein expression microarrays. The advent of high throughput methods allows to measure dozens of thousands or even millions features on different levels like DNA / RNA / protein. And nowadays in many large scale studies scientists use data from mRNA seq to assess the state of transcriptome, protein microarrays to asses the state of proteome and DNA-seq / bisulfide methylation to assess genome / methylome.</p>
<p><strong>Research methodology.</strong> Genetic signatures are widely used in different applications, among others: for assessing genes that differentiate cells that are chemo resistant vs. cells that are not, assess the stage of cell pluripotency, define molecular cancer subtypes. For example, in database Molecular Signatures Database v5.0 one can find thousands of gene sets - genetic signatures for various conditions. There are signatures that characterize some cancer cells, pluripotent cells and other groups. But they usually contain relatively small number of genes (around 100), results with them are hard to replicate and they are collection of features that were found significant when independently tested. In most cases signatures are derived from measurements of the same type. Like signatures based of expression of transcripts based on data from microarrays or RNA-seq, or methylation profile or DNA variation. We are proposing a very different approach. First we are going to use machine-learning techniques to create large collections of signatures. Such signatures base on ensembles of small sub-signatures, are more robust and usually have higher precision. Then out of such signatures we are going to develop methodology for meta-signatures, that integrate information from different types of data (transcriptome, proteome, genome). Great examples of such studies are: Progenitor Cell Biology Consortium (PCBC) and The Cancer Genome Atlas (TCGA) studies. For thousands of patients in different cohorts (for PCBC cohorts based on stemness phenotype, for TCGA based on cancer type) measurements of both mRNA, miRNA, DNA and methylation profiles are available. New, large datasets require new methods that take into account high and dense structure of dependencies between features. The task that we are going to solve is to develop methodology that will create genetic signatures that integrate information from different levels of cell functioning. Then we are going to use data from TCGA and PBCB project to assess the quality of proposed methodology. As a baseline we are going to use following methodologies: DESeq, edgeR (for mRNA), casper (for lternative splicing), MethylKit (for RRBS data) and RPPanalyzer for protein arrays.</p>
<p>Here is the skeleton for our approach: (1) Use ensembles in order to building a genetic signature. The first step would be to use random forests to train a new signature. Ensembles of sub-signtures are build on bootstrap subsamples and they votes if given sample fit given signature or not. (2) In order to improve signatures we are going to consider various normalization of raw counts. We start with log and rank transformation. (3) In order to improve the process of training an ensemble we are going to use pre-filtering of genes. (4) Another approach is to use Bayesian based methods, that may incorporate the expert knowledge, like belief-based gaussian modelling</p>
<p><strong>Research project impact.</strong> Genetic profiling is more and more important and has number of application starting from basic classification up to personalized medicine in which patients are profiled against different signatures. Existing tools for genetic signatures have many citations. This we assume that the methodology for integrated genetic profiling will be a very useful for many research groups. It is hard to overestimate the impact of better genetic profiling on medicine. Moreover we build a team of people with knowledge in cancer genetic profiling</p>
<p><strong>Work on this project is financially supported from the OPUS grant 2016/21/B/ST6/02176 funded by Polish National Science Centre (NCN).</strong></p>
</div>
</div>
</div>
<a href ="/" class="mobile-logo"><svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M28.6799 1.36802V28.68H1.31982V1.32004H28.6799V1.36802ZM30 0H0V30H30V0Z" fill="black"/><path d="M21.6357 21.0469V9.04688H22.9556V21.0469H21.6357Z" fill="black"/><path d="M18.5639 9.04688V21.0469H17.2439V11.0629L13.4039 21.0469H12.2039L8.36395 11.0629V21.0469H7.04395V9.04688H8.95195L12.8039 19.0789L16.6559 9.04688H18.5639Z" fill="black"/></svg></a>

<script>
  const topButtonHtml =
    `<div id="top-button-wrapper" onclick="scrollPageToTop()"><svg id="top-button" width="59" height="96" viewBox="0 0 59 96" fill="none"
     xmlns="http://www.w3.org/2000/svg"
     style="opacity: 0">
    <path d="M19.984 75.8284V84.2844H18.416V75.8284H15.35V74.5124H23.05V75.8284H19.984ZM28.9945 84.4524C28.3785 84.4524 27.8419 84.3404 27.3845 84.1164C26.9365 83.8831 26.5632 83.5518 26.2645 83.1224C25.9752 82.6931 25.7605 82.1658 25.6205 81.5404C25.4805 80.9151 25.4105 80.2011 25.4105 79.3984C25.4105 78.6051 25.4805 77.8958 25.6205 77.2704C25.7605 76.6358 25.9752 76.1038 26.2645 75.6744C26.5632 75.2451 26.9365 74.9184 27.3845 74.6944C27.8419 74.4611 28.3785 74.3444 28.9945 74.3444C29.6105 74.3444 30.1425 74.4611 30.5905 74.6944C31.0479 74.9184 31.4212 75.2451 31.7105 75.6744C32.0092 76.1038 32.2285 76.6358 32.3685 77.2704C32.5085 77.8958 32.5785 78.6051 32.5785 79.3984C32.5785 80.2011 32.5085 80.9151 32.3685 81.5404C32.2285 82.1658 32.0092 82.6931 31.7105 83.1224C31.4212 83.5518 31.0479 83.8831 30.5905 84.1164C30.1425 84.3404 29.6105 84.4524 28.9945 84.4524ZM28.9945 83.1504C29.6945 83.1504 30.1892 82.8891 30.4785 82.3664C30.7679 81.8344 30.9125 81.1158 30.9125 80.2104V78.5724C30.9125 77.6764 30.7679 76.9671 30.4785 76.4444C30.1892 75.9124 29.6945 75.6464 28.9945 75.6464C28.2945 75.6464 27.7999 75.9124 27.5105 76.4444C27.2212 76.9671 27.0765 77.6764 27.0765 78.5724V80.2244C27.0765 81.1204 27.2212 81.8344 27.5105 82.3664C27.7999 82.8891 28.2945 83.1504 28.9945 83.1504ZM35.7651 84.2844V74.5124H39.4191C40.3524 74.5124 41.0617 74.7738 41.5471 75.2964C42.0417 75.8191 42.2891 76.5378 42.2891 77.4524C42.2891 78.3671 42.0417 79.0858 41.5471 79.6084C41.0617 80.1311 40.3524 80.3924 39.4191 80.3924H37.3331V84.2844H35.7651ZM37.3331 79.0904H39.1811C40.1517 79.0904 40.6371 78.6424 40.6371 77.7464V77.1444C40.6371 76.2578 40.1517 75.8144 39.1811 75.8144H37.3331V79.0904Z" fill="black"/>
    <path d="M0.999908 30.2843L29.2842 2.00006L57.5685 30.2843" stroke="black" stroke-width="2"/>
    <path d="M29.2842 58.5686V2.00006" stroke="black" stroke-width="2"/>
</svg></div>`

  function scrollPageToTop () {
    window.scrollTo({top: 0, left: 0, behavior: 'smooth'})
    jQuery('.body-inner').animate({scrollTop: 0}, 'fast');
  }

  function handleMenu() {
    window.sessionStorage.clear()

    if (window.innerWidth < 900){
      const summary = window.document.getElementsByClassName("with-summary")
      if (summary && summary[0]) {
        summary[0].classList.remove('with-summary')
      }
    } else if (document.getElementsByClassName("book")) {
      document.getElementsByClassName("book")[0].classList.add('with-summary')
    }
  }

  window.addEventListener('DOMContentLoaded', (event) => {
    document.querySelector(".page-inner section").innerHTML += topButtonHtml

    handleMenu()

    setTimeout(()=>{
      if (window.innerHeight < window.document.getElementsByClassName("page-inner")[0].clientHeight - 10) {
        document.getElementById("top-button").style.opacity = 1
      }
    }, 2000)
  })

  window.addEventListener('resize', (event) => {
    handleMenu()
  })

  window.addEventListener('load', (event) => {
    const searchInput = document.getElementById('search-box')

    if (searchInput) {
      searchInput.setAttribute('placeholder', 'Search...')

      searchInput.parentElement.innerHTML += `<svg class="input-clear" width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg"><line x1="15.6729" y1="0.53033" x2="1.53077" y2="14.6725" stroke="black" stroke-width="1.5"/><line x1="15.6122" y1="14.6724" x2="1.47011" y2="0.530286" stroke="black" stroke-width="1.5"/></svg>
    `
    }
  });

</script>
            </section>

          </div>
        </div>
      </div>
<a href="papers.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="thesis-proposals.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook/js/app.min.js"></script>
<script src="libs/gitbook/js/clipboard.min.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script src="libs/gitbook/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
